static/app/components/assigneeSelectorDropdown.tsx

```diff
@@ -423,10 +417,7 @@ export default function AssigneeSelectorDropdown({
     const memberOptions = {
       value: '_members',
       label: t('Members'),
-      options:
-        memList?.map(member =>
-          makeMemberOption(member.id, member.name || member.email)
-        ) ?? [],
+      options: memList?.map(member => makeMemberOption(member)) ?? [],
```

Comment:
memList should always be defined
```suggestion
      options: memList.map(member => makeMemberOption(member)),
```